R/create.R

```diff
@@ -261,15 +261,20 @@ create_future_rt <- function(future = c("latest", "project", "estimate"),
 #'
 #' # using breakpoints
 #' create_rt_data(rt_opts(use_breakpoints = TRUE), breakpoints = rep(1, 10))
+#'
+#' # using random walk
+#' create_rt_data(rt_opts(rw = 7), breakpoints = rep(1, 10))
 #' }
 create_rt_data <- function(rt = rt_opts(), breakpoints = NULL,
                            delay = 0, horizon = 0) {
+
   # Define if GP is on or off
   if (is.null(rt)) {
```

Comment:
I've been thinking we should actually surface what settings are applied when `rt = NULL`. We only say that they can set it to NULL but we could add what settings are used when that is done. What do you think?

Reply:
yes I agree. A new issue though?

Reply:
I've created a new issue.